### PR TITLE
Quiz 1 Solutions

### PR DESCRIPTION
**Pull request by PaladinOreo**
Here are the solutions I came up with. If there are any errors or things that should be corrected, just let me know!
- Justice

### CODE REVIEW

File: docs/exercises_with_solutions.md [hunk 1]
```diff
@@ -2033,7 +2033,10 @@ They were given at week 4 and 7.
 
 ## Quiz 1
 
-#. (3 pts) Give three keywords.
+#. (3 pts) List three keywords.
+
+<details><summary>Solution</summary>Keywords are words that have special meanings and cannot be used as identifiers in your program. Examples of C# Keywords include int, string, if, while, void, else, bool, etc.</details>
```
Comment (aubertc) [inline docs/exercises_with_solutions.md:2038]: Please, use `this formatting` (with back-ticks, "`") for keywords.

File: docs/exercises_with_solutions.md [hunk 2]
```diff
@@ -2044,6 +2047,15 @@ They were given at week 4 and 7.
     - `test_train`{.text}
     - `_myIdentifier`{.text}
 
+<details><summary>Solution</summary>
+	```
```
Comment (aubertc) [inline docs/exercises_with_solutions.md:2051]: Maybe remind of the "rule", or why the others are not valid?

File: docs/exercises_with_solutions.md [hunk 3]
```diff
@@ -2054,13 +2066,36 @@ They were given at week 4 and 7.
...
 #. (4 pts) Write a statement that would display, "Hi Mom!" (*with* the quotes) followed by a new line on the screen.
+
+<details><summary>Solution</summary>```Console.WriteLine("\"Hi Mom!\"\n")```</details>
+
 #. (5 pts) Write a series of statements that would 
     #. declare an `int` variable called "myAge",
     #. assign your age to that variable, 
     #. display "My age is ", the value of the "myAge" variable, a period, and finally a new line.
+	
+<details><summary>Solution</summary>
```
Comment (aubertc) [inline docs/exercises_with_solutions.md:2087]: I _think_ this should be indented, to be part of the list.

File: docs/exercises_with_solutions.md [hunk 4]
```diff
@@ -2054,13 +2066,36 @@ They were given at week 4 and 7.
...
+int myAge;
+myAge = 21;
+Console.WriteLine("My age is " + myAge + ".\n");
+```
+</details>
+	
 #. (Bonus) Give examples of situations where the adage "_Spaces and new lines don't matter in programs_" is actually erroneous.
 
+<details><summary>Solution</summary>Spaces and newlines matter when they are used in string data, as whitespace in strings is formatted exactly how it's typed. Whitespace also matters inbetween words: words in C# must have at least one space between them in order to be compiled correctly (e.g. 'static void Main()' and 'int days = 7'). If there were no spaces in either of the examples, neither of them would compile.</details>
+
```
Comment (aubertc) [inline docs/exercises_with_solutions.md:2098]: Excellent. Maybe add that they matter for in-line comment?

```
// My comment
int x;
x = 10;
```

If you remove the first newline, the program would not compile.

**Review by aubertc — CHANGES_REQUESTED** [review]
Please, try to adjust and correct, @PaladinOreo . If you are not sure how to make edits to your pull request, please let me know.

**Review by PaladinOreo — COMMENTED** [review]
File edited with the suggestions added to it!

**Pull request merged**